design-system-docs/src/_component_examples/_radio_group/additional_attributes.erb

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
I'm not sure if/how to lint those files or if we should add a linting command later on

Reply:
We could maybe use something like [erb-lint](https://github.com/Shopify/erb-lint) but I don't know if the front matter would confuse things. I'm inclined to not worry about linting for the docs site, especially as we want designers to be able to easily edit things.